src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,49 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.FindBy;
+
+
+public class HomePage extends BasePage {
+    //private final By userAvatarIcon = By.cssSelector("img.avatar");
+    //private final By playlistInput = By.cssSelector("[name='name']");
+    //private final By alert =By.cssSelector("div.success.show");
+    //private final By playlist= By.xpath("//*[@id='playlists']/ul/li[3]");
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.